### DISCUSSION TITLE
Adding Zeek logs from external hosts

### DISCUSSION THREAD

**Discussion by MarkStanhope** [Security-Onion-Solutions/securityonion#3747] [Unsupported Versions]
I am trying to add some zeek logs from a number of linux hosts on my network, is this possible? 
To make the issue a little more complex some of the hosts are running on Raspberry's which filebeat does not directly support. 
I am happy to move the logs and do a daily update from another host or even from the security onion itself if that is the simplest solution. 
Thanks in advance. 
Mark

**Comment by TOoSmOotH** [comment]
I would try to use filebeat where possible and make sure you copy the same tags in the current file beat config from the normal sensors. As far as the rpi you will need to use syslog or something like it to send the data. That would probably need a parser for it as well.
